internal/capability/operator_cleanup.go

```diff
@@ -6,24 +6,24 @@ import (
 	"github.com/opdev/opcap/internal/operator"
 )
 
-func (ca *CapAudit) OperatorCleanUp() error {
+func (ca *capAudit) OperatorCleanUp() error {
 
 	// delete subscription
-	err := ca.Client.DeleteSubscription(context.Background(), ca.Subscription.Name, ca.Namespace)
+	err := ca.client.DeleteSubscription(context.Background(), ca.subscription.Name, ca.namespace)
```

Comment:
Can be combined with next line.

Reply:
I did all of them on commit 0f94b3e. @bcrochet  let me know if that's what you had in mind.